services/todo-legacy/package-lock.json

```diff
@@ -0,0 +1,946 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "dependencies": {
+    "@loopback/context": {
+      "version": "4.0.0-alpha.9",
+      "resolved": "https://registry.npmjs.org/@loopback/context/-/context-4.0.0-alpha.9.tgz",
+      "integrity": "sha1-iVCXFWjks14qacHQkaJ7acqh1ms="
+    },
+    "@loopback/core": {
+      "version": "4.0.0-alpha.9",
+      "resolved": "https://registry.npmjs.org/@loopback/core/-/core-4.0.0-alpha.9.tgz",
+      "integrity": "sha1-E07D3hILi3yB9KAr4xVvfu/p6AQ=",
+      "dependencies": {
+        "@loopback/context": {
+          "version": "4.0.0-alpha.9",
+          "resolved": "https://registry.npmjs.org/@loopback/context/-/context-4.0.0-alpha.9.tgz",
+          "integrity": "sha1-iVCXFWjks14qacHQkaJ7acqh1ms="
+        },
+        "@loopback/openapi-spec": {
+          "version": "4.0.0-alpha.6",
+          "resolved": "https://registry.npmjs.org/@loopback/openapi-spec/-/openapi-spec-4.0.0-alpha.6.tgz",
+          "integrity": "sha1-N9H3KLEPosXC46+f/aZHCsbUICc="
+        }
+      }
+    },
+    "@loopback/repository": {
+      "version": "4.0.0-alpha.3",
+      "resolved": "https://registry.npmjs.org/@loopback/repository/-/repository-4.0.0-alpha.3.tgz",
+      "integrity": "sha1-KIcUDZZphtyfm8MZB5B8V+JDjRA="
+    },
+    "@loopback/testlab": {
+      "version": "4.0.0-alpha.4",
+      "resolved": "https://registry.npmjs.org/@loopback/testlab/-/testlab-4.0.0-alpha.4.tgz",
+      "integrity": "sha1-5oXJaqpcaKH4B2cSNe4ONpP6e/E=",
+      "dev": true
+    },
+    "@types/bluebird": {
+      "version": "3.5.5",
+      "resolved": "https://registry.npmjs.org/@types/bluebird/-/bluebird-3.5.5.tgz",
+      "integrity": "sha512-aBVzCAzfyApU0gg36QgCpJixGtYwuQ4djrn11J+DTB5vE4OmBPuZiulgTCA9ByULgVAyNV2CTpjjvZmxzukSLw=="
+    },
+    "@types/http-errors": {
+      "version": "1.5.34",
+      "resolved": "https://registry.npmjs.org/@types/http-errors/-/http-errors-1.5.34.tgz",
+      "integrity": "sha1-1qVvJde5XdBwR2gL+CVjLil5aBU="
+    },
+    "@types/lodash": {
+      "version": "4.14.67",
+      "resolved": "http://apim-ci1.rtp.raleigh.ibm.com:8080/@types%2flodash/-/lodash-4.14.67.tgz",
```

Comment:
I think this should point to `registry.npmjs.org`, see my previous comment. There may be other package-lock entries pointing to our internal repository, I won't be looking for and commenting on them.